esp-hal/src/spi/slave.rs

```diff
@@ -464,20 +461,30 @@ where
         self.enable_dma();
 
         reset_dma_before_load_dma_dscr(reg_block);
+
         tx.prepare_transfer_without_start(self.dma_peripheral(), false, ptr, len)?;
+        #[cfg(esp32s2)]
+        tx.start_transfer()?;
 
-        self.clear_dma_interrupts();
         reset_dma_before_usr_cmd(reg_block);
 
-        // On the esp32, all full-duplex transfers are single, and all half-duplex
+        // Note: On the esp32, all full-duplex transfers are single, and all half-duplex
```

Comment:
I guess if we're not supporting the esp32, we don't need this comment and the other ones like it?

Reply:
I wanted to leave it there for future reference if someone breve enough will want to add ESP32 support, removed.